pkg/buildclient/client.go

```diff
@@ -31,11 +32,15 @@ type WebSocketDialer func(ctx context.Context, urlStr string, requestHeader http
 
 func Stream(ctx context.Context, cwd string, streams *streams.Output, dialer WebSocketDialer,
 	creds CredentialLookup, build *apiv1.AcornImageBuild) (*v1.AppImage, error) {
-	conn, _, err := dialer(ctx, wsURL(build.Status.BuildURL), map[string][]string{
+	conn, response, err := dialer(ctx, wsURL(build.Status.BuildURL), map[string][]string{
```

Comment:
Yep, good catch.

Reply:
This will cause panic if response.body is nil. Can you only read and close the body when it is not nil? just to program safely. For example, if you hit the network error, I think resp.body will be nil

Reply:
From the Go documentation:
```
The http Client and Transport guarantee that Body is always
non-nil, even on responses without a body or responses with
a zero-length body.
```

Reply:
nvm, ignore my comment